src/ServiceControl.Audit/Infrastructure/Settings/Settings.cs

```diff
@@ -44,6 +43,31 @@ public Settings(string serviceName, string transportType = null, string persiste
             EnableFullTextSearchOnBodies = SettingsReader<bool>.Read("EnableFullTextSearchOnBodies", true);
         }
 
+        void LoadAuditQueueInformation()
+        {
+            AuditQueue = SettingsReader<string>.Read("ServiceBus", "AuditQueue", "audit");
+
+            if (string.IsNullOrEmpty(AuditQueue))
+            {
+                throw new Exception("ServiceBus/AuditQueue requires a value to start the instance");
+            }
+
+            IngestAuditMessages = SettingsReader<bool>.Read("ServiceControl", "DisableAuditQueueIngestion", false);
```

Comment:
I think it would be best if the setting key and assigned property share the same name:
```suggestion
            DisableAuditQueueIngestion= SettingsReader<bool>.Read("ServiceControl", "DisableAuditQueueIngestion", false);
```

Reply:
Yet it needs to match https://github.com/Particular/ServiceControl/pull/3798 on the release branch, correct?

Either way if the name is "Ingest…" then the default should not be false, and if it's "Disable…" then the rest of the code needs to invert.

Reply:
I think proposal makes sense. I'll adjust the naming on both PRs.